aspnetcore/blazor/javascript-interop.md

```diff
@@ -101,6 +101,31 @@ The sample app includes a component to demonstrate JavaScript interop. The compo
 1. The `showPrompt` function accepts user input (the user's name), which is HTML-encoded and returned to the component. The component stores the user's name in a local variable, `name`.
 1. The string stored in `name` is incorporated into a welcome message, which is passed to a JavaScript function, `displayWelcome`, which renders the welcome message into a heading tag.
 
+## Call a void JavaScript function
```

Comment:
This seems a bit verbose to me. Shouldn't this just be a single sentence or two? I.e. "To call void returning JavaScript function call Invoke<object>. The returned value will be null" or something like that?